test/verify/check-machines-dbus

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+import unittest
+
+from testlib import *
+import machineslib
+
+
+class TestMachinesDBus(machineslib.TestMachines):
+
+    def detectLibvirtDbus(self):
+        m = self.machine
+        namesList = m.execute("dbus-send --system --dest=org.freedesktop.DBus "
+                              "--type=method_call "
+                              "--print-reply /org/freedesktop/DBus "
+                              "org.freedesktop.DBus.ListActivatableNames")
+
+        if ('org.libvirt' not in namesList):
+            raise unittest.SkipTest("libvirt-dbus is not available");
```

Comment:
Might be a bit more obvious to do something like `busctl introspect org.libvirt /` (or whichever its main object path is), and just see if it fails. But this is okay as well, and just nitpicking.

Reply:
I tired this, but I can't find a way to use cockpit dbus proxy call without specifying the interface but only the object path.
I would like to use this:
```proxy = client.proxy([interface, path], [options])```
but I should pass interface before path.

Reply:
Right, I actually just meant for the test here. As for `detectLibvirtProvider()`, if you want to change it there, don't use a proxy, that's too expensive. You just want to call a single method, like getting a property value or some `Ping()` like thing. I. e. use `client.call` there like you already do, just on some method on org.libvirt. In either case you have to specify an interface of the method or property, of course.

Reply:
Oh I misread that it was the js code. Done.

Reply:
No blanket `except:` please. Use `except subprocess.CalledProcessError:` to only catch this specific one. We don't want to silently hide `AttributeError` or something similarly nasty/silly.

Reply:
Done